sdk/storage/azure-storage-blob-changefeed/tests/test_change_feed.py

```diff
@@ -21,6 +21,7 @@
 
 from devtools_testutils.storage import StorageTestCase
 from settings.testcase import ChangeFeedPreparer
+from azure.core.credentials import AzureNamedKeyCredential
```

Comment:
I think this is left over and isn't need anymore?

Reply:
Good catch, unused import since we canned the changefeed test!